lib/tasks/convert_application_yml_to_proper_types.rake

```diff
@@ -0,0 +1,64 @@
+namespace :convert_application_yml_to_proper_types do
+  desc 'Take application.yml input file and convert to proper type'
+  task :convert, [:filename] => :environment do |t, args|
+    VALUE_TYPES = ['float', 'string', 'boolean', 'json', 'timestamp', 'integer']
+    filename = args[:filename]
+
+    if filename
+      content = ''
+      add_spaces = false
+      File.open(filename+'.tmp', 'w') do |newfile|
+        IO.foreach(filename) do |l|
+          add_spaces = true if l.include?('development:')
```

Comment:
Why only check `development:`? We need to worry about `test:` and `production:` stanzas too right?